compose/cli/command.py

```diff
@@ -57,7 +58,19 @@ def get_project(project_dir, config_path=None, project_name=None, verbose=False)
     api_version = os.environ.get(
         'COMPOSE_API_VERSION',
         API_VERSIONS[config_data.version])
-    client = get_client(verbose=verbose, version=api_version)
+    compose_tls_version = os.environ.get(
+        'COMPOSE_TLS_VERSION',
+        None)
+
+    tls_version = None
+    if compose_tls_version == "TLSv1":
+        tls_version = ssl.PROTOCOL_TLSv1
+    if compose_tls_version == "TLSv1_1":
+        tls_version = ssl.PROTOCOL_TLSv1_1
```

Comment:
IMO It should raise an exception if the user is trying to use an unsupported `tls_version`.